mapflow/functional/service/data_catalog.py

```diff
@@ -61,13 +67,30 @@ def get_mosaic_callback(self, response: QNetworkReply):
         mosaic = MosaicReturnSchema.from_dict(response.readAll().data())
         self.mosaics.update({mosaic.id: mosaic})
         self.mosaicsUpdated.emit()
+    
+    def update_mosaic(self):  
+        mosaic = self.selected_mosaic()
+        dialog = UpdateMosaicDialog(self.dlg)
+        dialog.accepted.connect(lambda: self.api.update_mosaic(mosaic_id=mosaic.id, mosaic=dialog.mosaic(), callback=self.update_mosaic_callback))
+        dialog.setup(mosaic)
+        dialog.deleteLater()
+        self.dlg.mosaicTable.clearSelection()
+        self.dlg.mosaicInfo.setText("Mosaic info")
+    
+    def update_mosaic_callback(self, response: QNetworkReply):
```

Comment:
Do we really need to send a new request after successful upadate? I think that it is a bit overhead, can we use the existing data + the updated row?